test/cypress/integration/24-event.createOrder.test.js

```diff
@@ -1,4 +1,5 @@
-describe('event.createOrder page', () => {
+// This needs to be converted to the new Page
+describe.skip('event.createOrder page', () => {
```

Comment:
Don't have the time to migrate it and didn't want to delete it. I can open an issue to convert it to the new page.

Reply:
Is `makes an order for tickets with VAT` test also using the old flow? I don't see the `/legacy` in there. Or maybe that's because `NEW_EVENTS` is not the default for E2E yet?

Reply:
That's right, it should work. We should not skip it.